package.json

```
@@ -134,6 +134,11 @@
             "maxNumberOfProblems": 100,
             "trace.server": "off"
           }
+        },
+        "terraform.rootModules": {
+          "scope": "resource",
+          "type": "array",
```

Found this example here: https://github.com/golang/vscode-go/blob/93a25eb84ff1aad58cee3eff9126bb09c857d7ca/package.json#L866-L874
```suggestion
          "type": "array",
          "items": {
            "type": "string"
          },
          "default": []          
```